pkg/runtime/dynamicload.go

```diff
@@ -0,0 +1,116 @@
+package runtime
+
+import (
+	"path/filepath"
+
+	"github.com/dapr/dapr/pkg/components"
+	"github.com/fsnotify/fsnotify"
+	"github.com/pkg/errors"
+
+	componentsV1alpha1 "github.com/dapr/dapr/pkg/apis/components/v1alpha1"
+)
+
+func (a *DaprRuntime) watchPathForDynamicLoading() {
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.Errorf("unable to create watcher for dynamic components, dynamic loading will not be supported: %s", err)
+		return
+	}
+	defer watcher.Close()
+	err = watcher.Add(a.runtimeConfig.Standalone.ComponentsPath)
+	if err != nil {
+		log.Errorf("unable to watch components directory: %s , err: %s , dynamic loading will not be supported", a.runtimeConfig.Standalone.ComponentsPath, err)
```

Comment:
nit: .... dynamic loading of **components** ....

Reply:
updated!